app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
```

Comment:
@apeksha2697  please remove this file from PR

Reply:
done